src/pretix/base/pdf.py

```diff
@@ -939,6 +939,12 @@ def _draw_textarea(self, canvas: Canvas, op: OrderPosition, order: Order, o: dic
         if o['italic']:
             font += ' I'
 
+        # Since pdfmetrics.registerFont is global, we want to make sure that no one tries to sneak in a font, they
+        # should not have access to.
+        if font not in list(get_fonts(order.event, pdf_only=True).keys()) + ['Open Sans']:
```

Comment:
Running the signal for every text box we render does not sound terribly efficient

Reply:
Adding a 30 second cache; open to discuss a longer runtime.

Reply:
I was thinking about a refactor that does this once per render, not once per text field on that render. With the cache, we'll still need to hit the cache via network on every text field?

Reply:
Moved to __init__